bin/bi-local

```diff
@@ -229,7 +229,7 @@ phx-server)
     do_phx_server
     ;;
 int-test)
-    do_int_test
+    do_int_test "${args[@]}"
```

Comment:
Allow passing args to the underlying `bix ex int-test` command to allow testing single modules or whatever.